htdocs/thumbs.php

```diff
@@ -161,14 +170,13 @@
                 break;
         }
 
-        if ($im == '') {
+        if (!isset($im)) {
```

Comment:
Wouldn't 'empty' better?

Reply:
It has content ore it wasn't declared :)

Reply:
-.- can be merged